src/sentry/tasks/store.py

```diff
@@ -331,30 +331,40 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None,
         manager = EventManager(data)
         manager.save(project_id)
     except HashDiscarded:
-        tsdb.incr(
-            tsdb.models.project_total_received_discarded,
-            project_id,
-            timestamp=to_datetime(start_time) if start_time is not None else None,
-        )
+        increment_list = [
+            (tsdb.models.project_total_received_discarded, project_id,),
```

Comment:
Trailing comma after `project_id` is stylistically inconsistent with the rest of these tuples below

Reply:
oops good call